wifiphisher/macmatcher.py

```diff
@@ -1,91 +1,123 @@
-#pylint: skip-file
 """
 This module was made to match MAC address with vendors
 """
 
-from constants import *
+import wifiphisher.constants as constants
 
 
 class MACMatcher(object):
     """
     This class is using Organizationally Unique Identifiers (OUIs)
-    in order to match MAC addresses of devices to its vendord
+    in order to match MAC addresses of device to its vendor
```

Comment:
I would change it to "This class handles Organizationally Unique Identifiers (OUIs). The original data comes from http://standards.ieee.org/regauth/oui/oui.txt"